prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Syntax</title>
+</head>
+<h1 style="color: darkcyan;">My favorite Star Wars character is Baby Yoda!</h1>
+<img src="https://www.indiewire.com/wp-content/uploads/2020/11/baby-yoda-grogu.png" alt="Baby Yoda" width="650" height= "370">
+<p style="color: cornflowerblue;">Baby Yoda is small but powerful and doesn't realize it yet</p>
```

Comment:
Oop! Be sure to put visible contend inside of a `<body>` element

```html
<body>
  <h1 style="color: darkcyan;">My favorite Star Wars character is Baby Yoda!</h1>
  <img src="https://www.indiewire.com/wp-content/uploads/2020/11/baby-yoda-grogu.png" alt="Baby Yoda" width="650" height= "370">
  <p style="color: cornflowerblue;">Baby Yoda is small but powerful and doesn't realize it yet</p>
</body>
```

Reply:
Whoops! Thanks for letting me know